front/src/pages/Home.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Home: React.FC = () => {
```

Comment:
up to my knowledge it's better without using `React.FC`:

https://medium.com/raccoons-group/why-you-probably-shouldnt-use-react-fc-to-type-your-react-components-37ca1243dd13
https://github.com/facebook/create-react-app/pull/8177